### PR TITLE
Add References

### PR DESCRIPTION
**Pull request by prithaupadhyay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,3 +170,11 @@ This is Float32 extension of [Google's Sample Image Classifier for Android](http
 			<li><a href="https://github.com/aashutoshrathi">Aashutosh Rathi</a></li>
 		</ul>
 </details>
+
+##References
```
Comment (aashutoshrathi) [inline README.md:174]: Space after `##` else it won't work.

File: README.md [hunk 2]
```diff
@@ -168,3 +170,11 @@ This is Float32 extension of [Google's Sample Image Classifier for Android](http
...
+
+<ul>
+	<li><a href="<a href="https://www.tensorflow.org/tutorials/image_retraining">">Model Retraining and Transfer Learning</li>
+	<li><a href="<a href="https://codelabs.developers.google.com/codelabs/tensorflow-for-poets-2-tflite/#0">">Generate TFLite file</li>
+	
+</ul>
```
Comment (aashutoshrathi) [inline README.md:180]: Also, for listing MD doesn't support HTML completely.
For listing do it like
```md
- [Item 1](link1)
- [Item 2](link2)
```

**Comment by anshumanv** [comment]
@prithaupadhyay please do the requested changes as suggested by @aashutoshrathi  :bowing_man:

**Comment by prithaupadhyay** [comment]
Did the required changes and sent a new PR #4

**Pull request closed without merging**